eng/scripts/Language-Settings.ps1

```diff
@@ -244,7 +244,7 @@ function ValidatePackagesForDocs($packages) {
     # Get value for variables outside of the Foreach-Object scope
     $scriptRoot = "$using:scriptRoot"
     $workingDirectory = "$using:tempDirectory"
-    return ."$scriptRoot\validate-docs-package.ps1" -Package $_ -WorkingDirectory $workingDirectory
+    return ."$scriptRoot\validate-docs-package.ps1" -Package $_ -DocValidationImageId "$using:ImageId" -WorkDirectory $workingDirectory 
```

Comment:
What does `using` do here? I'm not familiar with that PS scope. Is the idea that this will come in from the global scope entry point? Should we instead pass that through as a parameter? What if this value is not set?

Reply:
Synced offline. We have to use keyword "$using:" to access the var outside of the `ForEach-Object -Parellel`.

It passed through the `Update-DocsMsPackage.ps1`, and we default it empty, so there is no case that ImageId not set at all.

Reply:
Will kickoff a test on fallback logic which no image id case.